lib/extensions/postgres_cdc_rls/subscriptions_checker.ex

```diff
@@ -79,16 +79,7 @@ defmodule Extensions.PostgresCdcRls.SubscriptionsChecker do
       Subscriptions.delete_multi(state.conn, ids)
```

Comment:
@abc3 we should only delete from `ets` (`:ets.delete(tid, pid)`) once `Subscriptions.delete_multi(state.conn, ids)` is successful.

Reply:
Good call! I've added this to tasks and will do it in another PR